app_dart/lib/src/request_handlers/luci_status.dart

```diff
@@ -81,6 +88,78 @@ class LuciStatusHandler extends RequestHandler<Body> {
     return Body.empty;
   }
 
+  Future<void> _processCompleted({
```

Comment:
Does `_taskCompleted(...)` make more sense in this situation?

Reply:
Ahh this is ambiguous.  The ideas is this method should process completed jobs, it's a bad name.  I'll update and document.